clients/admin-ui/src/features/config-wizard/AuthenticateRuntimeForm.tsx

```diff
@@ -19,7 +19,9 @@ import ScannerLoading from "./ScannerLoading";
 const AuthenticateRuntimeForm = () => {
   const dispatch = useAppDispatch();
   const toast = useToast();
-  const { data, error: queryError } = useGetScanResultsQuery();
+  const { data, error: queryError } = useGetScanResultsQuery({
```

Comment:
Interesting that this is a GET request. And by interesting I mean "yikes". It kicks off a process that will, among other things, create ClassifyInstances, right? Unless I'm missing something.

Not a UI code problem, if that's case, just raised my eyebrows. Maybe the component shouldn't be called "Authenticate", though, if were really kicking it off just by entering this step.

Reply:
ahh right, that's a good point. we had planned to change it over to PUT as part of https://github.com/ethyca/fidesctl-plus/issues/195 (which will write scan history), but you're right that we should do it now that we're writing classify instances. @SteveDMurphy 's being a hero and said he'll change it to a PUT as part of https://github.com/ethyca/fidesctl-plus/pull/304, and then I'll update the frontend in https://github.com/ethyca/fides/pull/1752

Yeah, it probably shouldn't be called Authenticate. it was the easiest way to fit it into the existing onboarding flow though. I think https://github.com/ethyca/fides/issues/1514 will make the "steps" more true to each flow, and then this component won't have to be stuffed into "Authenticate". I'll rename it for now though 👍